lib/ViewModels/DataCatalogTabViewModel.js

```diff
@@ -38,4 +40,32 @@ DataCatalogTabViewModel.prototype.showInfo = function(item) {
     CatalogItemInfoViewModel.open('ui', item);
 };
 
+DataCatalogTabViewModel.prototype.clickEnabled = function(item) {
+    ga('send', 'event', 'dataSource', 'clickItem', item.name);
+
+    if (!item.isEnabled && defined(item.initialMessage) && item.initialMessage.length > 0) {
+        PopupMessageViewModel.open('ui', {
+            title: 'Data Set Information',
+            message: '<div>'+ item.initialMessage +'</div>'
+        });
+        item.initialMessage = undefined;
+    }
+
+    item.toggleEnabled();
+};
+
+DataCatalogTabViewModel.prototype.clickOpen = function(item) {
+    ga('send', 'event', 'dataSource', 'clickOpen', item.name);
+
+    if (!item.isOpen && defined(item.initialMessage) && item.initialMessage.length > 0) {
+        PopupMessageViewModel.open('ui', {
```

Comment:
Because this is done here (rather than being something initiated by the `CatalogItem` itself, it will only happen when the catalog item is enabled via the UI.  E.g. it won't show up when the catalog item is enabled via a share link or by any other programmatic mechanism.  Is this what we want?